wikidp/controllers/auth.py

```diff
@@ -0,0 +1,87 @@
+"""Methods for user authentication and authorization."""
```

Comment:
Is this left over from somwhere, I'm not sure this should be here? This looks like duplication from the oauth module and is probably not what's needed?

Reply:
Apologies for the confusion, meant to call this out in a PR comment before adding reviewers!

This is just moving the auth methods that aren't route-specific to the controller module so we can reuse them in other places in the codebase. For example, in the write api route, I needed a check that the user is authenticated and the function that grabs the WDLogin, so this controller module serves a shared entry-point. (In this PR `routes/oauth.py` now imports these methods.)